### DISCUSSION TITLE
Exception: Unable to find javac

### DISCUSSION THREAD

**Discussion by LongxingTan** [castorini/pyserini#1941] [Q&A]
Thanks for the nice tool.

Could you please give me a hint how to solve this? I didn't realize that I need to set java env?

I try to run a eval method in pyserini
```
python -m pyserini.eval.trec_eval -c -mrecip_rank ...
```

and in pyclass.py it import jnius
Then it raise the exception as above:   
```
File "/root/.conda/envs/open/lib/python3.10/site-packages/jnius/env.py", line 351, in get_jdk_home
    raise Exception('Unable to find javac')
```

**Comment by cuichenxu** [comment]
Got the same question, I installed the lastest java jdk, the question is solved.
